posts/exercism-is-the-official-go-mentoring-platform.md

```diff
@@ -0,0 +1,14 @@
+So I'm really proud to announce that as of today, Exercism is becoming the official mentoring platform for GoBridge and the Go Developer Network (GDN).
+
+GoBridge's mission is to enable minorities in tech to use Go as a tool to learn and teach programming and, ultimately, to empower underrepresented groups in tech to help increase diversity in the Go community.
+That fits perfectly with our mission of providing opportunity through the development of programming, and so we're really excited to be able to launch this partnership.
+GoBridge and the GDN will be promoting Exercism across their networks, and we'll be hoping for a large influx of both Go mentors and students.
+
+As part of this partnership, GoBridge is also donating to Exercism to help fund improvements to our interactivity, specifically allowing us to lay the foundations of student-mentor interactions **within** the new Exercism editor.
+This is something we're really excited about developing, and will form the center of Exercism Teams, which we'll be launching in 2021.
+
+We've put a special landing page up for this partnership, which you can access at [https://exercism.org/partners/gobridge](https://exercism.org/partners/gobridge).
+If you've not tried Go before, it's a very interesting language and we're got a really [solid track](https://exercism.org/tracks/go) on Exercism, with some great [Learning Exercises](https://exercism.org/tracks/go/concepts).
```

Comment:
```suggestion
If you've not tried Go before, it's a very interesting language and we've got a [solid track](https://exercism.org/tracks/go) on Exercism, with some great [Learning Exercises](https://exercism.org/tracks/go/concepts).
```

Too much very and really etc otherwise, imo